liminal/core/config/config.py

```diff
@@ -0,0 +1,146 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+import traceback
+
+from liminal.core.config.defaults import hyperliminal, default_configs
+from liminal.core.util import dict_util
+from liminal.core.util import files_util
+
+
+class ConfigUtil:
+    """
+    Load and enrich config files under configs_path.
+    """
+    __HYPERLIMINAL = "hyperliminal"
+    __PIPELINES = "pipelines"
+    __SUPER = "super"
+    __TYPE = "type"
+    __SUB = "sub"
+    __SERVICES = "services"
+    __TASKS = "tasks"
+    __PIPELINE_DEFAULTS = "pipeline_defaults"
+
+    def __init__(self, configs_path):
+        self.configs_path = configs_path
+        self.config_files = files_util.load(configs_path)
+        self.hyperliminal = hyperliminal.HYPERLIMINAL
```

Comment:
wouldn't it be better to name hyperliminal something with a more "standard" association?
like archetype?

Reply:
@aviemzur ?

Reply:
Hyperliminal is to be a specific file for defaults to enable local running of applications without special configurations.
The name was selected because of another concept of `superliminal` or `super` to introduce inheritence, while all liminal systems also inherit from the `hyperliminal` file at their topmost level